platform/android/detect.py

```diff
@@ -134,7 +134,7 @@ def mySpawn(sh, escape, cmd, args, env):
 		env['SHLINKFLAGS'][1] = '-shared'
 		env['SHLIBSUFFIX'] = '.so'
 	elif (os.name=="nt"):
-		gcc_path=gcc_path+"/windows-x86_64/bin" #this may be wrong
```

Comment:
Also I want to point out that last time I've visited NDK download page there was a mixup, link to 32bit binary pointed to 64bit binary, and vice versa. You just assume that you have downloaded 64bit version and that folder must be called "windows" now, but it's not.